### PR TITLE
[azure-kinect-sensor-sdk] Fix azure-kinect-sensor-sdk[tool] build error

### LINKED ISSUE #13715: [azure-kinect-sensor-sdk] build failure
**Host Environment**
- OS: Windows 10

**To Reproduce**
./vcpkg install azure-kinect-sensor-sdk[tool]

The "tool" feature of azure-kinect-sensor-sdk relies on the imgui[bindings] which does not exist. I think what happened is imgui port got patched and no longer has [bindings] but all the individual [xxx-binding] features.

I'm not sure reviving [bindings] is the way to go or adding all the binding options to azure-kinect-sensor-sdk is the way to go.

### PR DESCRIPTION
**Pull request by LilyWangL**
**Describe the pull request**

- What does your PR fix? Fixes #13715
Due to PR #12329 fix the features of `imgui`, `azure-kinect-sensor-sdk[tool]` build failed with the error: `Package imgui does not have a bindings feature`.

### CODE REVIEW

**Comment by LilyWangL** [comment]
@RT222 Could you review this PR? Thanks.

**Comment by RT222** [comment]
Reintroducing the Imgui source files in the include path doesn't seem to be necessary. One of the main goal of my PR was to build the bindings, which should remove the need to manually include Imgui cpp files.

Without a single change to the imgui port, you should be able to build azure-kinect-sensor-sdk[tool] with the following patch :

fix-dependency.patch
```
diff --git a/tools/k4aviewer/CMakeLists.txt b/tools/k4aviewer/CMakeLists.txt
index 6ab38d9..f31f570 100644
--- a/tools/k4aviewer/CMakeLists.txt
+++ b/tools/k4aviewer/CMakeLists.txt
@@ -54,6 +54,8 @@ include_directories(
     ${CMAKE_CURRENT_LIST_DIR}
 )
 
+find_package(GLEW REQUIRED)
+
 set(EXTERNAL_LIBRARIES
     k4a::k4a
     k4a::k4arecord
@@ -65,6 +67,7 @@ set(EXTERNAL_LIBRARIES
     glfw
     ${OPENGL_LIBRARIES}
     unofficial::gl3w::gl3w
+    GLEW::GLEW
 )
 
 # On Windows, we need to call into setupapi to get USB container ID information
diff --git a/tools/k4aviewer/k4asoundio_util.h b/tools/k4aviewer/k4asoundio_util.h
index c9c2718..011a76e 100644
--- a/tools/k4aviewer/k4asoundio_util.h
+++ b/tools/k4aviewer/k4asoundio_util.h
@@ -16,7 +16,7 @@
 // This disables that behavior.
 //
 #define NOMINMAX
-#include <soundio.h>
+#include <soundio/soundio.h>
 
 // Project headers
 //
 
```

File: ports/azure-kinect-sensor-sdk/fix-dependency.patch [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
 --- a/tools/k4aviewer/CMakeLists.txt
 +++ b/tools/k4aviewer/CMakeLists.txt
 @@ -35,6 +35,8 @@ set(SOURCE_FILES
      k4awindowset.cpp
      perfcounter.cpp
      ${CMAKE_CURRENT_BINARY_DIR}/version.rc
-+    ${IMGUI_EXTERNAL_PATH}/imgui_impl_glfw.cpp
-+    ${IMGUI_EXTERNAL_PATH}/imgui_impl_opengl3.cpp
++    ${IMGUI_EXTERNAL_PATH}/../imgui_impl_glfw.cpp
++    ${IMGUI_EXTERNAL_PATH}/../imgui_impl_opengl3.cpp
```
Comment (RT222) [inline ports/azure-kinect-sensor-sdk/fix-dependency.patch:10]: Manually adding imgui bindings sources isn't required anymore since #12329

**Review by RT2Code — CHANGES_REQUESTED** [review]

**Comment by LilyWangL** [comment]
@RT222 Thanks for your reply.

**Comment by jjpepper** [comment]
@LilyWangL any idea when this will be merged?  I just pulled and am still seeing the `Package imgui does not have a bindings feature` error.

**Review by JackBoosY — APPROVED** [review]

**Comment by BillyONeal** [comment]
Thanks for your contribution!

**Pull request merged**